newsroom/default_settings.py

```diff
@@ -117,7 +128,8 @@
 # Email settings
 MAIL_SERVER = os.environ.get('MAIL_SERVER') or 'localhost'
 MAIL_PORT = os.environ.get('MAIL_PORT') or 25
-MAIL_USE_SSL = os.environ.get('MAIL_USE_SSL') or False
+MAIL_USE_SSL = strtobool(os.environ.get('MAIL_USE_SSL')) or False
+MAIL_USE_TLS = strtobool(os.environ.get('MAIL_USE_TLS')) or False
```

Comment:
I would rather import all these `MAIL*` settings from superdesk so we have it in sync